src/Bicep.Core/Emit/PlaceholderParametersJsonWriter.cs

```diff
@@ -10,9 +10,9 @@
 
 namespace Bicep.Core.Emit
 {
-    public class ParametersFileTemplateWriter : TemplateWriter
+    public class PlaceholderParametersJsonWriter : TemplateWriter
```

Comment:
>Placeholder [](http://example.com/codeflow?start=17&length=11)

Same question about this placeholder class as well.

Reply:
Placeholder == the existing 'generate params file' logic (generates a params file with blank values from a `.bicep` file)

Reply:
Talked with @cqthyy earlier today, I think she's going to add comments to explain the purposes.